napari/_qt/qt_main_window.py

```diff
@@ -489,15 +489,16 @@ def __init__(self, viewer: 'Viewer', *, show: bool = True):
         get_settings().appearance.events.theme.connect(self._update_theme)
 
         self._add_viewer_dock_widget(
-            self._qt_viewer.dockConsole, tabify=False, menu=self.window_menu
+            self._qt_viewer.dockConsole,
+            tabify=False,
```

Comment:
```suggestion
            self._qt_viewer.dockConsole, tabify=False,
```

Reply:
Amended and then let black do its thing.